### PR TITLE
Run pylint against Python 3.5

### PR DESCRIPTION
**Pull request by lukesneeringer**
It turns out @jonparrott was completely right and I was completely wrong, and pylint does need to run against Python 3.5. I did not notice the failure because we were ignoring its output (which we still are, which I intend to fix as soon as is feasible).

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -38,6 +38,8 @@ deps =
   sphinx
   -r{toxinidir}/docs/requirements-docs.txt
 commands = make -C docs html latex
+whitelist_externals =
+  make
```
Comment (lukesneeringer) [inline tox.ini]: This just gets rid of a tox warning that complains about `make` not being part of the virtualenv.

Comment (geigerj) [inline tox.ini]: Weird. Could you put a comment to that effect?

Comment (lukesneeringer) [inline tox.ini]: I can? It's [standard tox behavior][1], though.

  [1]: http://tox.readthedocs.io/en/latest/example/basic.html

Comment (geigerj) [inline tox.ini]: Ah, I see -- I had missed the fact that `make` was being used in the command. Either way is fine.

**Pull request merged**